modules/gapi/include/opencv2/gapi/gcompoundkernel.hpp

```diff
@@ -65,6 +65,16 @@ template<typename U> struct get_compound_in<cv::GArray<U>>
     }
 };
 
+template<typename U> struct get_compound_in<cv::GOpaque<U>>
```

Comment:
GArray was here, so I think we can handle GOpaque too. But not sure if was needed